client/extensions/woocommerce/state/sites/orders/reducer.js

```diff
@@ -78,12 +78,13 @@ export function isQueryLoading( state = {}, action ) {
  * @return {Object}        Updated state
  */
 export function isUpdating( state = {}, action ) {
+	const orderId = isNumber( action.orderId ) ? action.orderId : JSON.stringify( action.orderId );
```

Comment:
Maybe use `isFinite` instead off `isNumber` for security against `Infinity`, `-Infinity`, and `NaN`.

Reply:
Thanks for the tip - I doubt we'll need to worry about `Infinity`, but something that catches `NaN` sounds useful 🙂